pulsar/internal/connection.go

```diff
@@ -288,7 +288,7 @@ func (c *connection) doHandshake() bool {
 
 	if cmd.Connected == nil {
 		c.log.Warnf("Failed to perform initial handshake - Expecting 'Connected' cmd, got '%s'",
-			cmd.Type)
+			cmd.Error.GetMessage())
```

Comment:
Please change the original message since with this change it wouldn't make sense anymore:

```
c.log.Warnf("Failed to establish connection with broker: '%s'",
```